--- Source/LibRawLite/internal/defines.h	2015-03-08 13:37:22.000000000 -0700
+++ Source/LibRawLite/internal/defines.h	2016-04-26 21:39:59.271455100 -0700
@@ -55,7 +55,9 @@
 #include <sys/utime.h>
 #include <winsock2.h>
 #pragma comment(lib, "ws2_32.lib")
+#if _MSC_VER < 1900
 #define snprintf _snprintf
+#endif
 #define strcasecmp stricmp
 #define strncasecmp strnicmp
 #line 85 "dcraw/dcraw.c"
--- Source/LibTIFF4/tif_config.h	2015-03-01 16:50:58.000000000 -0800
+++ Source/LibTIFF4/tif_config.h	2016-04-26 21:39:58.360005800 -0700
@@ -81,7 +81,9 @@
 #endif // BYTE_ORDER

 #ifdef _WIN32
+#if _MSC_VER < 1900
 #define snprintf _snprintf
+#endif
 #define lfind _lfind
 #endif // _WIN32

