commit 234cd8ce9cc29e8d64aeadab697469c8266f193c
Author: Walter Gray <wgray@leapmotion.com>
Date:   Thu Mar 8 18:56:36 2018 -0800

    test

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5599fda..c8518e4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,7 +47,7 @@ set(FlatBuffers_Compiler_SRCS
   src/idl_gen_cpp.cpp
   src/idl_gen_general.cpp
   src/idl_gen_go.cpp
-  src/idl_gen_js.cpp
+  #src/idl_gen_js.cpp
   src/idl_gen_php.cpp
   src/idl_gen_python.cpp
   src/idl_gen_fbs.cpp
diff --git a/src/flatc_main.cpp b/src/flatc_main.cpp
index 02d01c0..cbd51f6 100644
--- a/src/flatc_main.cpp
+++ b/src/flatc_main.cpp
@@ -71,16 +71,16 @@ int main(int argc, const char *argv[]) {
       flatbuffers::IDLOptions::kJava,
       "Generate Java classes for tables/structs",
       flatbuffers::GeneralMakeRule },
-    { flatbuffers::GenerateJS,       "-s", "--js", "JavaScript", true,
-      nullptr,
-      flatbuffers::IDLOptions::kJs,
-      "Generate JavaScript code for tables/structs",
-      flatbuffers::JSMakeRule },
-    { flatbuffers::GenerateJS,       "-T", "--ts", "TypeScript", true,
-      nullptr,
-      flatbuffers::IDLOptions::kTs,
-      "Generate TypeScript code for tables/structs",
-      flatbuffers::JSMakeRule },
+//    { flatbuffers::GenerateJS,       "-s", "--js", "JavaScript", true,
+//      nullptr,
+//      flatbuffers::IDLOptions::kJs,
+//      "Generate JavaScript code for tables/structs",
+//      flatbuffers::JSMakeRule },
+//    { flatbuffers::GenerateJS,       "-T", "--ts", "TypeScript", true,
+//      nullptr,
+//      flatbuffers::IDLOptions::kTs,
+//      "Generate TypeScript code for tables/structs",
+//      flatbuffers::JSMakeRule },
     { flatbuffers::GenerateGeneral,  "-n", "--csharp", "C#", true,
       nullptr,
       flatbuffers::IDLOptions::kCSharp,
